src/main/java/com/sarahhand/fractals/model/MandelbrotConfig.java

```diff
@@ -52,6 +56,15 @@ public ColorPalette getPalette() {
 		return palette;
 	}
 	
+	@JsonIgnore
+	public ColorScheme getColorScheme(){
+		return colorScheme;
+	}
+
+	public void setColorScheme(ColorScheme colorScheme){
```

Comment:
I don't think we should have a set method for this. We don't have set methods for any of the other properties.